src/containers/Chat/ChatMessages/ChatMessages.tsx

```
@@ -398,8 +405,21 @@ export const ChatMessages = ({ entityId, collectionId, phoneId }: ChatMessagesPr
 
         fetchMore({
           variables,
-          updateQuery: (prev: any, { fetchMoreResult }: any) =>
-            updateCacheQuery(prev, fetchMoreResult, entityId, collectionId, chatType),
+          updateQuery: (prev: any, { fetchMoreResult }: any) => {
+            if (fetchMoreResult.search && fetchMoreResult.search.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)